### DISCUSSION TITLE
Getting random error when pressing `l` on the main screen and updating does not work

### DISCUSSION THREAD

**Discussion by KoinBanks** [LazyVim/LazyVim#4554] [Q&A] [answered]
When pressing `l` on a main screen, I get this error
![image](https://github.com/user-attachments/assets/33d7777e-e0ae-47d6-a832-5a23f44aa16f)

 `...l/share/nvim/lazy/lazy.nvim/lua/lazy/manage/task/git.lua:90: branch 'main' not found`
 
 I started getting this after system update I think (?)
 
It seems it cycles between that error and these errors:
 
 ```
 ...l/share/nvim/lazy/lazy.nvim/lua/lazy/manage/task/git.lua:90: attempt to concatenate local 'v' (a table value)
 ...l/share/nvim/lazy/lazy.nvim/lua/lazy/manage/task/git.lua:90: tag 'v12.42.0' not found
```

What should I do? How do I debug this?

My config: https://github.com/iFarmGolems/nvim

This configuration works as expected on different machine.

On both machines I'm using arch linux in WSL.

Thanks in advance.

**Comment by KoinBanks** [comment]
When I press `U` (update) on the update screen:

![image](https://github.com/user-attachments/assets/d1c4d416-de81-4bc0-ba77-73e805e95361)

**Reply by dpetka2001** [reply]
Probably some glitch with git. I can update just fine on my end. Try again later.

**Reply by KoinBanks** [reply]
Found the fix: https://github.com/LazyVim/LazyVim/discussions/4554#discussioncomment-10877658

**Comment by KoinBanks** [comment, marked as answer]
I found fix: https://stackoverflow.com/a/77546153/4972544

I had to delete ~/.local/share/nvim/ and run nvim again.
